server/src/main/resources/nacos-config.txt

```diff
@@ -23,7 +23,14 @@ store.file.max-global-session-size=512
 store.file.file-write-buffer-cache-size=16384
 store.file.flush-disk-mode=async
 store.file.session.reload.read_size=100
-store.db.driver_class=com.mysql.jdbc.Driver
-store.db.url=jdbc:mysql://localhost:3306/seata_demo
+store.db.db-type=mysql
+store.db.url=jdbc:mysql://localhost:3306/fescar_demo
```

Comment:
why do you change the datasource name? seata-demo maybe ok?

Reply:
> why do you change the datasource name? seata-demo maybe ok?

done